qa/standalone/ceph-helpers.sh

```diff
@@ -120,6 +120,13 @@ function setup() {
     teardown $dir || return 1
     mkdir -p $dir
     mkdir -p $(get_asok_dir)
+    if [ $(ulimit -n) -le 1024 ];then
```

Comment:
please add a space after `;` to be more consistent with the other part of this file.

Reply:
fixed.